dfloat11_model_loader.py

```diff
@@ -77,7 +77,7 @@ def load_dfloat11_model_advanced(self, dfloat11_model_name, cpu_offload, cpu_off
             pin_memory=pin_memory,
         )
 
-        if df11_type == "Chroma" and custom_modelpatcher:
+        if custom_modelpatcher:
```

Comment:
The `CustomChromaModelPatcher` class is supposed to be for experimental LoRA loading for Chroma models only. If you check the code, my custom overridden class patches the `state_dict()` method of the diffusion model to return a state_dict corresponding to Chroma, which is not ideal in the general case.